src/ethereum_test_tools/spec/blockchain_test.py

```diff
@@ -284,6 +294,7 @@ def make_blocks(
                 previous_head=head,
                 chain_id=chain_id,
                 eips=eips,
+                last_block=(block == self.blocks[-1]),
```

Comment:
We might need to rethink a bit what we are trying to test with the ForkchoiceUpdated here because if we are only trying to set it to the expected head of the chain, then we can simply use the `head` variable which already always has the expected head of the chain (if the last block is invalid, then the head is `self.blocks[-2]` instead of `self.blocks[-1]` for example).